src/main/java/tutoraid/model/lesson/Students.java

```diff
@@ -57,10 +57,10 @@ public void addStudent(Student student) {
      */
     public void removeStudent(Student student) {
         requireNonNull(student);
+        System.out.println(students.contains(student));
```

Comment:
Was this intentionally left in or maybe you used for debugging?

Reply:
Ah thank you for notifying me this, I was debugging. Will delete it now.